feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go

```diff
@@ -528,6 +528,11 @@ func TestEstablish(t *testing.T) {
 	t.Logf("Start DUT interface Config")
 	configureDUT(t, dut)
 
+	// Configure Network instance type on DUT
```

Comment:
Should be a period at the end. That's supposed to be true throughout, although I see it's not quite today

Reply:
Done.